### DISCUSSION TITLE
how to use mPDF in the Yetiforce PDF generation

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#16556] [How to]
Hi,

I installed mPDF using composer. The installation completed without any error. 

But when i check the PDF template, it displayed only YetiForcePDF engine. 

Could you please guide me to install the mPDF to use it in the PDF template. If there any documentation avaiable for integrating the mPDF with Yetiforce kindly share the same.

Thanking you.

with regards,
Rajasekaran.

**Comment by mariuszkrzaczkowski** [comment]
Add a new driver https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/app/Pdf/Drivers
It is best to load additional libraries in the custom directory as the second composer
